2/contrib/s2i/run

```diff
@@ -79,9 +87,9 @@ function install_plugins() {
   fi
   
   if [ "$(ls -A /opt/openshift/plugins 2>/dev/null)" ]; then
-    mkdir -p ${JENKINS_HOME}/plugins
     echo "Copying $(ls /opt/openshift/plugins | wc -l) Jenkins plugins to ${JENKINS_HOME} ..."
-    cp -r /opt/openshift/plugins/* ${JENKINS_HOME}/plugins/
+    # since install_plugins populates this, the files already end in jpi
+    for FILENAME in /opt/openshift/plugins/* ; do cp --remove-destination $FILENAME ${JENKINS_HOME}/plugins; done
```

Comment:
couldn't still use cp -r?

Reply:
it occurred to me that there could be non-jpi files here (leftover lock files); decided to be more precise, thinking any overhead from a few more `cp` calls is neglible

Reply:
Same comments as made above apply here.  Given the move from cp -r, did you mean to do do FILENAME in .../*.jpi here?

Reply:
yeah, about to contradict myself here ... i moved to single file cp to avoid unavoided things, but now realize that at the time avoided the prefix in case we switched from jpi to hpi

it should be jpi, based on how the download works

I'll make it specific - thanks for the catch